packages/ui-extender/src/actions/menuAction/menuAction.jsx

```diff
@@ -144,7 +144,7 @@ const reducer = (state, action) => {
             return {
                 ...state,
                 loadingItems: state.loadingItems.filter(f => f !== action.item),
-                loadedItems: action.isVisible !== false && state.loadedItems.indexOf(action.item) === -1 ? [...state.loadedItems, action.item] : state.loadedItems
```

Comment:
action.isVisible is optional and true by default , why changing that behaviour ?

Reply:
because sometimes it's undefined and the old condition make it does in the first block and add the action into the menu then the visibility condition of the whole menu is true because there is items 
in Jcontent the other conditon is not satisfied so disabled=false

Reply:
isVisible undefined means that the item is visible. Only isVisible=false makes it invisible - it was the same behavior for enabled, where only enabled=false makes it disabled